src/mahoji/lib/abstracted_commands/minionKill.ts

```diff
@@ -181,6 +181,9 @@ export async function minionKillCommand(
 		usersTask.currentTask !== null &&
 		usersTask.assignedTask.monsters.includes(monster.id);
 
+	let isUsingKonar = false;
+	if (isOnTask && usersTask.slayerMaster.id === 5) isUsingKonar = true;
```

Comment:
This should be changed to be the same as these:

```
usingCannon: !usingCannon ? undefined : usingCannon,
		cannonMulti: !cannonMulti ? undefined : cannonMulti,
```

and change the type to `isUsingKonar?: true;`, so its either undefined, or true.

This is just to not have unnecessary stuff in the data in the db.